src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -270,6 +270,13 @@ Some common problems after migration:
     * If you belonged to multiple organizations, it's possible to see a combination of old logins for unmigrated organizations along with the new user login.
     * Your email address may have multiple user records because you belong to multiple organizations or multiple authentication domains within the same organization.
 
+### Multi-tenant organization migration [#multi-tenant]
+
+If your New Relic organization was a [customer partnership](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partnership-accounts-users-subscriptions) (a complex multi-tenant structure) and you were migrated to the new user model, you may see a seeming increase in data ingest in NRQL queries you've created. Your ingested data hasn't actually changed but due to the migration, you may experience some discrepencies. 
+
+For details about how to create working queries, see [Usage queries](src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts#multitenant-ingest).
```

Comment:
@zuluecho9 I think you need to ditch the `src/content` in the URL. Also, I'd clarify the sentence like this:

`For details about how to create usage queries...`